mmeds/config.py

```diff
@@ -21,14 +21,18 @@
     STORAGE_DIR = Path(resources.__file__).parent.resolve()
     # If apache is trying to access apache's home directory for `mmeds_server_data` rather than your user account's home
     # you may need to hardcode the path here
+    # TODO: Add mmeds configuration options for DATABASE_DIR
     if gethostname() == 'fedora-y70':
         DATABASE_DIR = Path('/home/david') / 'mmeds_server_data'
+    elif gethostname() == 'Hal':
+        DATABASE_DIR = Path('/home/matt') / 'mmeds_server_data'
     else:
         DATABASE_DIR = Path().home() / 'mmeds_server_data'
     SERVER_PATH = 'http://localhost/myapp/'
     CSS_DIR = 'http://localhost/CSS/'
     IMAGE_PATH = str(CSS_DIR) + '/'
 else:
+    # TODO: Figure out if all these fp settings being set to wallad07 going to cause issues down the road.
```

Comment:
ah it should be `mmedsadmin` not `wallad07`. Not sure why it isn't

Reply:
ok, I'll add that to the TODO flag. Not sure I want to make the change myself since I don't know what impact it could have